collections/rpc-development-framework.yml

```diff
@@ -0,0 +1,17 @@
+ident: rpc-development-framework
+name: Rpc Development Framework
+name_cn: RPC开发框架
+slug: /rpc-development-framework
+items:
+  - https://github.com/apache/incubator-brpc
+  - https://github.com/grpc/grpc
+  - https://github.com/apache/dubbo
+  - https://github.com/apache/thrift
+  - https://github.com/twitter/finagle
+  - https://github.com/grpc/grpc-go
+  - https://github.com/grpc-ecosystem/grpc-gateway
+  - https://github.com/andeya/erpc
+  - https://github.com/TarsCloud/Tars
+  - https://github.com/elixir-grpc/grpc
+  - https://github.com/twitchtv/twirp
```

Comment:
@QYFabc ,I suggest taking a look at this: https://oss-compass.org/collection/remote-procedure-call

Reply:
OK